llvm-hs/src/LLVM/Internal/Operand.hs

```diff
@@ -837,8 +838,8 @@ instance EncodeM EncodeAST A.DILocalVariable (Ptr FFI.DILocalVariable) where
 
 instance EncodeM EncodeAST A.DITemplateParameter (Ptr FFI.DITemplateParameter) where
   encodeM p = do
-    name' <- encodeM (A.name (p :: A.DITemplateParameter)) :: EncodeAST (Ptr FFI.MDString)
-    ty <- encodeM (A.type' (p :: A.DITemplateParameter))
+    name' <- encodeM (DT.name (p :: A.DITemplateParameter)) :: EncodeAST (Ptr FFI.MDString)
+    ty <- encodeM (DT.type' (p :: A.DITemplateParameter))
```

Comment:
@cocreature, hi! I am not sure that I understand this part correctly. Can i ask for review?